### ISSUE TITLE
Manual theme?

### ISSUE THREAD

**Issue by Developer-Mike** [khmyznikov/pwa-install#16]
Is it possible to always use the white theme?

**Comment by khmyznikov** [comment]
@Developer-Mike can you describe your usecase? The idea of this component was to match look n feel of browser ui, do you have any missbehaviour here?

**Comment by Developer-Mike** [comment]
I have a white-themed website. So a black box looks a bit odd.

**Comment by khmyznikov** [comment]
> I have a white-themed website. So a black box looks a bit odd.

Yeah but the UI of the browser? For example, native android install box will show you in system/browser theme, not your webapp color.

**Comment by Developer-Mike** [comment]
I understand your arguments, but I don't want to trick the users into thinking it's from the browser (Like those scam websites).

**Comment by khmyznikov** [comment]
I see you point, it's not a trick but as you wish. I will think how this can be done without extra css.

**Comment by khmyznikov** [comment]
Every template have this [section ](https://github.com/khmyznikov/pwa-install/blob/5125865fbf8d9a5041268fc3f32244ab6bfbb4c8/src/templates/apple/styles-apple.scss#L479)for dark theme. Maybe it can be separated to own file and be optionally [included here](https://github.com/khmyznikov/pwa-install/blob/5125865fbf8d9a5041268fc3f32244ab6bfbb4c8/src/index.ts#L39) by flag but it's need to be explored, maybe you can help with it?

**Comment by Developer-Mike** [comment]
I'll look if I have time for it.
